carberretta/bot/cogs/meta.py

```diff
@@ -36,7 +36,26 @@ async def twitter_command(self, ctx: commands.bot.Context) -> None:
     @commands.command(name="patreon")
     async def patreon_command(self, ctx: commands.bot.Context) -> None:
         await ctx.send("https://patreon.carberra.xyz")
-
-
+        
+    @commands.command(name="plans", aliases=["plan"])
+    async def plans_command(self, ctx: commands.bot.Context) -> None:
+        await ctx.send("https://plans.carberra.xyz")
+        
+    @commands.command(name="facebook")
+    async def facebook_command(self, ctx: commands.bot.Context) -> None:
+        await ctx.send("https://facebook.carberra.xyz")
+        
+    @commands.command(name="lbry", aliases=["odysee"])
+    async def lbry_command(self, ctx: commands.bot.Context) -> None:
+        await ctx.send("https://lbry.carberra.xyz")
+    
+    @commands.command(name="donate")
+    async def donate_command(self, ctx: commands.bot.Context) -> None:
+        await ctx.send("https://donate.carberra.xyz")
```

Comment:
This should point to a new redirect I'm gonna set up for Streamlabs tips:
```suggestion
    @commands.command(name="donate", aliases=["tip"])
    async def donate_command(self, ctx: commands.bot.Context) -> None:
        await ctx.send("https://tip.carberra.xyz")
```

Reply:
commands.Context, not commands.bot.Context.

Reply:
is tip.carberra.xyz a working redirect site ? cus I cannot go to that link 😕